### PR TITLE
various improvements

### PR DESCRIPTION
**Pull request by bmwiedemann**

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -574,14 +545,17 @@ function confset()
     return 0
 }
 
+function macfunc()
```
Comment (jdsn) [inline scripts/mkcloud]: please also support lonleynodes with this function, they differ in one block

File: scripts/mkcloud [hunk 2]
```diff
@@ -605,8 +579,7 @@ function setuplonelynodes()
 function crowbar_register()
 {
     for i in $lonelynodeids; do
-        local i2=$( printf %02x $i )
-        local mac="52:54:77:88:77:$i2"
```
Comment (jdsn) [inline scripts/mkcloud]: change of behaviour: lonleynodes contain 88, regular nodes not

**Comment by jdsn** [comment]
:+1:

**Comment by rsalevsky** [comment]
+1

**Pull request merged**